src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -107,3 +134,39 @@ def init_sync_d_interface_tables():
         if_alias_map = dict(zip(if_name_map.keys(), if_name_map.keys()))
 
     return db_conn, if_name_map, if_alias_map, if_id_map, oid_sai_map, oid_name_map
+
+
+def init_sync_d_lag_tables(db_conn):
+    """
+    Helper method. Connects to and initializes LAG interface maps for SyncD-connected MIB(s).
+    :param db_conn: database connector
+    :return: tuple(lag_name_if_name_map, if_name_lag_name_map, oid_lag_name_map)
+    """
+    db_conn.connect(APPL_DB)
+
+    lag_entries = db_conn.keys(APPL_DB, b"LAG_TABLE:*")
```

Comment:
this function seems to be only executed at the init time. this may not be complete at certain point of time. For example, the system has 4 port channels, and at the time when you tries to get the LAG_TABLE, you only see 3.